### PR TITLE
feat: adding alert css and documentation page

### PR DESCRIPTION
**Pull request by ultronjs**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (t007rushi) [inline .vscode/settings.json:1]: you can remove/untrack or can use .gitignore as no need of this file in remote repo

File: css/abstracts/_variables.css [hunk 2]
```diff
@@ -4,6 +4,11 @@
     /* --secondary: #4B3F72; */
     --secondary-color:#119DA4;
 
+
+    --primary_background: #d1b5f7;
+    /* --secondary_background: #897cb4; */
```
Comment (t007rushi) [inline css/abstracts/_variables.css:9]: I think you can reconsider naming & comments for easy in reading & future use

**Review by t007rushi — CHANGES_REQUESTED** [review]
Just one more thing site looks good just use a different extension for code snippet just suggestion

**Review by Raghav888 — APPROVED** [review]
Looks good

**Pull request merged**